FEATURES.md

```diff
@@ -30,14 +30,20 @@ If you want to install a particular chart, simply hover the pointer over the cha
 ![Repository7](images/screenshot_repository7.png)
 
 # Installed Releases list
-This informs us about the release version we are currently using. You can utilize the filter option to refine your search for a specific release by employing namespaces and freetext.
-![Releases1](images/screenshot_release.png)
+A release is an instance of your selected chart running on your Kubernetes Cluster. That means every time that you install a Helm chart there, it creates a new release or instance that coexists with other releases without conflict. You can filter releases based on namespaces or search for release names 
+![Releases](images/screenshot_release.png)
 
-A revision is linked to a release to track the number of updates/changes that release encounters.
+The squares represent k8s resources installed by the release. Hover over each square to view a tooltip with details. Yellow indicates "pending," green signifies a healthy state, and red indicates an unhealthy state.
+![Releases1](images/screenshot_release1.png)
+
+This indicates the list of releases and the corresponding chart versions.
```

Comment:
"In list as a whole" what you want can you be more specific
First I describe what is release and the purpose of release then I tell you can search release using a text filter and namespace and then I explain individually row as you told me to do

Reply:
Here's the structure of product:

Level 1: Page
Level 2: List of installed releases, filter by namespace, filter by text
Level 3: Each item of the list displays: name, status, age etc.

Right now this logic is not followed. You talk about level 2, then one item of level 3, then level 2 again, then level 3 again. It would confuse the reader a lot.